dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/Continuous/FallingEdgeDetection.mo

```diff
@@ -0,0 +1,42 @@
+within Dynawo.NonElectrical.Blocks.Continuous;
+
+/*
+* Copyright (c) 2022, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools for power systems.
+*/
+
+block FallingEdgeDetection "Output y is true, if the input u has fallen to a lower value since the previous sample"
```

Comment:
What is the difference with the FallingEdge blocks in the MSL Logical and/or MathBoolean ?

Reply:
Is it the tS ?

Reply:
As you saw, the MSL blocks take a boolean input.